app/assets/javascripts/mapping.js

```diff
@@ -69,16 +28,24 @@ var MRConfig = (function () {
 (function() {
     var map;
     $(document).ready(function() {
-        var road_layer = new L.TileLayer('http://otile{s}.mqcdn.com/tiles/1.0.0/map/{z}/{x}/{y}.png', {
+        var osm_layer = new L.TileLayer('http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+            maxZoom: 18,
+            attribution: "&copy; <a href=\'http://openstreetmap.org\'> OpenStreetMap</a> contributors"
+        }),
+        road_layer = new L.TileLayer('http://otile{s}.mqcdn.com/tiles/1.0.0/map/{z}/{x}/{y}.png', {
             maxZoom: 18,
             subdomains: ['1', '2', '3', '4'],
             attribution: 'Tiles Courtesy of <a href="http://www.mapquest.com/" target="_blank">MapQuest</a>. Map data (c) <a href="http://www.openstreetmap.org/" target="_blank">OpenStreetMap</a> contributors, CC-BY-SA.'
         }),
-        satellite_layer = new L.TileLayer('http://otile{s}.mqcdn.com/tiles/1.0.0/sat/{z}/{x}/{y}.png', {
+        mapquest_layer = new L.TileLayer('http://otile{s}.mqcdn.com/tiles/1.0.0/sat/{z}/{x}/{y}.png', {
```

Comment:
That is good to know, currently I was just playing around with it to see what was available and what could be done. I think when we are closer to the finished product we can nail down exactly what this should look like.